parkings/models/parking.py

```diff
@@ -42,17 +42,13 @@ def registration_number_like(self, registration_number):
         return self.filter(normalized_reg_num=normalized_reg_num)
 
 
-class Parking(TimestampedModelMixin, UUIDPrimaryKeyMixin):
+class AbstractParking(TimestampedModelMixin, UUIDPrimaryKeyMixin):
     VALID = 'valid'
     NOT_VALID = 'not_valid'
 
-    region = models.ForeignKey(
-        Region, null=True, blank=True, on_delete=models.SET_NULL,
-        related_name='parkings', verbose_name=_("region"),
-    )
+    region = models.ForeignKey(Region, null=True, blank=True, on_delete=models.SET_NULL, verbose_name=_("region"))
```

Comment:
I guess this is just for removing the related_name? It would be better to not touch the styling while removing this single argument.

I think the related_name  needs to be added to the Parking model then again.

Reply:
Oh, there was a default_related_name. Didn't know that such thing exists. Nice. :)